### ISSUE TITLE
Build error: can't download yarn-1.22.10.tar.gz

### ISSUE THREAD

**Issue by jvermillard** [eclipse/leshan#1428]
### Version(s)

ca1e6bc585ae95332dd5f87b4d35f0d0727590fd

### Which components

leshan-server-demo

### What happened

https://github.com/eclipse/leshan/actions/runs/4453950979/jobs/7843751932

```
[INFO] Downloading https://sourceforge.net/projects/yarn.mirror/files/v1.22.10/yarn-v1.22.10.tar.gz to /home/runner/.m2/repository/com/github/eirslett/yarn/1.22.10/yarn-1.22.10.tar.gz

Error:  Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.12.1:install-node-and-yarn (install node and yarn) on project leshan-server-demo: Could not download Yarn: Got error code 404 from the server. -> [Help 1]
```

This version of yarn is not available there: https://sourceforge.net/projects/yarn.mirror/files/

**Comment by sbernard31** [comment]
Yep we use source forge mirror because of : https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1032

But I didn't know that they don't keep all version of yarn, this will make more complicate, if we want to rebuild old version of Leshan.

So maybe we should  only use source forge for jenkins (by creating a new profile on just change the URL passing parameters ?)

**Comment by sbernard31** [comment]
> So maybe we should only use source forge for jenkins (by creating a new profile on just change the URL passing parameters ?)

I created a PR for this : #1430

**Comment by sbernard31** [comment]
Fixed by PR #1429 and #1430
